src/when.js

```diff
@@ -0,0 +1,34 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * Tests the final argument by passing it to the given predicate function.
+ * If the predicate is satisfied, the function will return the result
+ * of calling the `whenTrueFn` function with the same argument. If the predicate
+ * is not satisfied, the argument is returned as is.
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @see R.ifElse, R.unless
+ * @sig (a -> Boolean) -> (a -> a) -> a -> a
+ * @param {Function} pred       A predicate function
+ * @param {Function} whenTrueFn A function to invoke when the `condition`
+ *                              evaluates to a truthy value.
+ * @param {*}        x          An object to test with the `pred` function and
+ *                              pass to `whenTrueFn` if necessary.
+ * @return {Function} A new function that will process the `whenTrue` function
+ *                    depending upon the result of the `condition` predicate.
```

Comment:
You're correct. (I should have just scrapped the entire docs and started again when I changed the signature...)

I've pushed up the changed docs.